inst/stan/functions/gaussian_process.stan

```diff
@@ -2,35 +2,56 @@
 // see here for details: https://arxiv.org/pdf/2004.11408.pdf
 real lambda(real L, int m) {
   real lam;
-  lam = ((m*pi())/(2*L))^2;
+  lam = ((m * pi())/(2 * L))^2;
   return lam;
 }
+
 // eigenfunction for approximate hilbert space gp
 // see here for details: https://arxiv.org/pdf/2004.11408.pdf
 vector phi(real L, int m, vector x) {
   vector[rows(x)] fi;
-  fi = 1/sqrt(L) * sin(m*pi()/(2*L) * (x+L));
+  fi = 1/sqrt(L) * sin(m * pi()/(2 * L) * (x + L));
   return fi;
 }
+
 // spectral density of the exponential quadratic kernal
 real spd_se(real alpha, real rho, real w) {
   real S;
-  S = (alpha^2) * sqrt(2*pi()) * rho * exp(-0.5*(rho^2)*(w^2));
+  // S = (alpha^2) * sqrt(2 * pi()) * rho * exp(-0.5 * (rho^2) * (w^2));
+  S = 2.506628 * alpha * rho * exp(-0.5 * (rho^2) * (w^2));
```

Comment:
does precomping the constants here help? My impression was stan was already doing this.